### PR TITLE
Update build/bundle. Bump to 0.0.5

### PR DESCRIPTION
**Pull request by doktordirk**
- update dependencies
- fontawesome moved back to index.html
- added bluebird for ie support
- removed unneeded export task. our bundle exports allready
- split bundle into vendor and app. some parallel loading will make startup faster. and i like it's cleaniness
- more cleaning and clarifications
- fixed bundling

### CODE REVIEW

**Comment by RWOverdijk** [comment]
@doktordirk Ooooooops. I merged something and now this has to be rebased.

**Comment by doktordirk** [comment]
i'll manage

File: README.md [hunk 1]
```diff
@@ -20,12 +20,16 @@ The component demonstrates:
...
+- [aurelia-datatable](https://github.com/SpoonX/aurelia-datatable)
+- [aurelia-pager](https://github.com/SpoonX/aurelia-pager)
+- [aurelia-form](https://github.com/SpoonX/aurelia-form)
 
 ## Build tasks
 
-### Bundle
-Creates a production-ready build in the `dist/` directory.
+### `gulp bundle`
+Creates a production-ready build in the `dist/` directory. Needs admin rights for symlinking.
```
Comment (RWOverdijk) [inline README.md]: on windows*

File: build/bundles.js [hunk 2]
```diff
@@ -2,18 +2,43 @@ module.exports = {
   "bundles": {
     "scripts/app-build": {
       "includes": [
+        "[**/*.js]",
```
Comment (RWOverdijk) [inline build/bundles.js:5]: Does this exclude jspm_packages?

File: dist/index.html [hunk 3]
```diff
@@ -13,10 +16,24 @@
   <i class="fa fa-spinner fa-spin"></i>
 </div>
 
+<!-- The bluebird version is locked at 3.4.1 in the package.json file to keep this from breaking -->
+<!-- We include bluebird to bypass Edge's very slow Native Promise implementation. The Edge team -->
+<!-- has fixed the issues with their implementation with these fixes being distributed with the  -->
+<!-- Windows 10 Anniversary Update on 2 August 2016. Once that update has pushed out, you may    -->
+<!-- consider removing bluebird from your project and simply using native promises if you do     -->
+<!-- not need to support Internet Explorer.                                                      -->
+<script src="jspm_packages/npm/bluebird@3.4.1/js/browser/bluebird.min.js"></script>
```
Comment (RWOverdijk) [inline dist/index.html:25]: Won't (shouldn't) work when bundled.

File: src/index.html [hunk 4]
```diff
@@ -3,6 +3,9 @@
 <head>
   <title>Aurelia</title>
   <meta charset="utf-8">
+  <!-- FontAwesome is used for the spinner. To make it available as fast as possible, we load it here. -->
+  <!-- The FontAwesome version is locked at 4.6.3 in the package.json file to keep this from breaking. -->
+  <link rel="stylesheet" href="jspm_packages/npm/font-awesome@4.6.3/css/font-awesome.min.css">
```
Comment (RWOverdijk) [inline src/index.html:8]: Won't (shouldn't) work when bundled.

File: src/scripts/page/paged/demo.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  <div class="panel panel-primary">
+    <div class="panel-heading">Database error</div>
+    <div class="panel-body">
+      <paged repository.bind="notARepository" data.bind="noData" error.bind="otherError">
+        <div repeat.for="todo of noData" if.bind="!otherError">
+          Nothing to see here.
+        </div>
+
+        <div class="alert alert-warning" if.bind="!!otherError">
+          Yeah, that didn't work!
```
Comment (RWOverdijk) [inline src/scripts/page/paged/demo.html]: Translate

File: src/scripts/page/paged/demo.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      </paged>
+    </div>
+  </div>
+
+  <div class="panel panel-primary">
+    <div class="panel-heading">Database error</div>
+    <div class="panel-body">
+      <paged repository.bind="notARepository" data.bind="noData" error.bind="otherError">
+        <div repeat.for="todo of noData" if.bind="!otherError">
+          Nothing to see here.
```
Comment (RWOverdijk) [inline src/scripts/page/paged/demo.html]: Translate

File: src/scripts/page/paged/demo.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        <div repeat.for="todo of data" if.bind="!error">
+          <div class="row">
+            <div class="col-xs-4">${todo.todo}</div>
+            <div class="col-xs-4">${todo.list.name}</div>
+            <div class="col-xs-4"><a href=mailto:${todo.user.username}>${todo.user.username}</a></div>
+          </div>
+        </div>
+
+        <div if.bind="!!error">
+          Yeah, that didn't work!
```
Comment (RWOverdijk) [inline src/scripts/page/paged/demo.html]: Translate

File: src/scripts/page/paged/demo.html [hunk 8]
```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="panel panel-primary">
+    <div class="panel-heading">Database from todo: page:2 limit:2</div>
```
Comment (RWOverdijk) [inline src/scripts/page/paged/demo.html]: What's this?

File: src/scripts/page/association-select/demo.html [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+        <association-select
+          repository.bind="todoRepository"
+          property="todo"
+          error.bind="todoError"
+          value.bind="todoId"
+          criteria.bind="listId ? {where:{list:listId}} : {}"
+        ></association-select>
+
+        <div class="alert alert-warning" if.bind="!!totError && !listError">
+          Error loading lists: ${todoError.statusText}
```
Comment (RWOverdijk) [inline src/scripts/page/association-select/demo.html]: translate

File: build/bundles.js (hunk 2 above)
Comment (doktordirk) [inline build/bundles.js:5]: yes. i think all that's not in other bundles or so. anyways. jspm_packages are not in app-build.js in the end

File: dist/index.html (hunk 3 above)
Comment (doktordirk) [inline dist/index.html:25]: as mentioned the needed jspm_packages get transfered to /dist

File: src/index.html (hunk 4 above)
Comment (doktordirk) [inline src/index.html:8]: see above

**Comment by doktordirk** [comment]
dist made with #https://github.com/SpoonX/aurelia-orm/pull/173

**Pull request merged**